libs/s25main/ai/aijh/AIPlayerJH.cpp

```diff
@@ -1105,14 +947,14 @@ MapPoint AIPlayerJH::FindPositionForBuildingAround(BuildingType type, const MapP
     {
         case BuildingType::Woodcutter:
         {
-            foundPos = FindBestPosition(around, AIResource::Wood, BUILDING_SIZE[type], 20, 11);
+            foundPos = FindBestPosition(around, AIResource::Wood, BUILDING_SIZE[type], 11, 20);
```

Comment:
can we name those magic constants somehow useful?

Reply:
I have no idea where they are coming from and why. Probably experimenting.
However the "11" as radius is used often, so we could name this at least.

Reply:
ok that would be great

Reply:
Done